listenbrainz/mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -414,6 +431,67 @@ def create_cache(mb_conn, mb_curs, lb_conn, lb_curs):
     log("mb metadata cache: done")
 
 
+def delete_rows(lb_curs, recording_mbids):
+    query = """
+        DELETE FROM mb_metadata_cache
+              WHERE recording_mbid IN %s
+    """
+    lb_curs.execute(query, (tuple(recording_mbids),))
+
+
+def config_postgres_join_limit(mb_curs):
+    # Because of the size of query we need to hint to postgres that it should continue to use joins
```

Comment:
```suggestion
    # Because of the size of query we need to hint to postgres that it should continue to
    # reorder JOINs in an optimal order. Without these settings, PG will take minutes to
    # execute the metadata cache query for even for 3-4 mbids. With these settings,
    # the query planning time increases by few milliseconds but the query running time
    # becomes instantaneous.
```